pegasus/sites/code.org/views/course_block.haml

```diff
@@ -1,9 +1,21 @@
-.courseblock-span10.courseblock-wide-large{style: "height: 200px; margin: 20px 80px 0 80px;"}
-  %a{href: url}
-    .courseblock-span4.imgspan
-      %img{src: image}
-    .courseblock-span6.heading
-      %h3= title
-      .smalltext= description
-    .bottombar.smalltext.trynow Try now!
-.clear{style: "margin-bottom: 20px;"}
+- teacherview ||= nil
+- if teacherview?
```

Comment:
does this work?  there is both `teacherview` and `teacherview?` in these two lines.